src/jit/compiler.cpp

```diff
@@ -10692,11 +10685,10 @@ void cNodeIR(Compiler* comp, GenTree* tree)
         chars += printf(" ");
         chars += cLeafIR(comp, tree);
     }
-    else if (op == GT_LEA)
+    else if !(op == GT_LEA) // Already dumped GT_LEA above.
```

Comment:
I am not familiar with the `!` outside the parentheses. Is this a new syntax in C++?

Reply:
I apologize, there were a few commits that should not have slipped in, but did. I fixed this!